### DISCUSSION TITLE
Resource linked-data webpage

### DISCUSSION THREAD

**Discussion by royassis** [apache/jena#2284] [Q&A] [answered]
Is it possible to generate a linked-data website with Apache Jena.

i.e. generate the web pages for the resources.
e.g. Like such: https://www.w3.org/1999/02/22-rdf-syntax-ns#type OR https://id.nlm.nih.gov/mesh/D000069544.html

If not, is there a tool for generating those?

**Comment by rvesse** [comment, marked as answer]
Not with Jena directly. Jena is primarily a set of building blocks for enabling RDF driven applications to be built.

Perhaps something like [LinkedDataHub](https://atomgraph.github.io/LinkedDataHub/linkeddatahub/docs/get-started/) which is built using Jena might be appropriate?

**Reply by royassis** [reply]
Thanks @rvesse.
Hope you don't mind that I'll ask some more questions.
How does Apache Jena Fuseki server fit in this ? If any

**Reply by namedgraph** [reply]
@royassis for example, LinkedDataHub persists RDF data in Fuseki (uses it as a database).

**Reply by rvesse** [reply]
> Thanks @rvesse.
> Hope you don't mind that I'll ask some more questions.
> How does Apache Jena Fuseki server fit in this ? If any 

It's effectively a database server. That it can have a minimalist UI is primarily a convenience for developers. What it really exists for is providing HTTP APIs to allow accessing RDF databases across multiple, potentially remote, clients/processes

**Reply by royassis** [reply]
In the link to [LDP Implementations](https://www.w3.org/wiki/LDP_Implementations) @ajs6f sent.  
There's a tagging to client/server. So is Apache Jena Fuseki similar to a LDP server by that logic ?

**Reply by namedgraph** [reply]
@royassis LDP is a different protocol than SPARQL, so yes you can say so.
However I would say LDP is the worst and most unnecessary spec in the RDF stack. What it was intended to do is covered by Linked Data and the [Graph Store Protocol](https://www.w3.org/TR/sparql11-http-rdf-update/) (which Fuseki supports since it's a part of SAPRQL 1.1).

**Reply by royassis** [reply]
Thank you.  
Is there a forum where I can ask more questions? What is the appropriate place?

**Reply by namedgraph** [reply]
@royassis Semantic Web mailing list: https://lists.w3.org/Archives/Public/semantic-web/
LinkedDataHub-specific mailing list: https://groups.io/g/linkeddatahub

**Comment by ajs6f** [comment]
You may also wish to use a [Linked Data Platform](https://www.w3.org/TR/ldp/) implementation for this work. It was specifically designed for such things. Several are based on or use Jena, and you can find a quick list [here](https://www.w3.org/wiki/LDP_Implementations).
